test/integration/concrete/lockup-base/create-with-timestamps/createWithTimestamps.t.sol

```diff
@@ -68,7 +68,13 @@ abstract contract CreateWithTimestamps_Integration_Concrete_Test is Integration_
         expectRevert_DelegateCall(callData);
     }
 
-    function test_RevertWhen_BrokerFeeExceedsMaxValue() external whenNoDelegateCall {
+    function test_RevertWhen_ShapeNameExceeds32Bytes() external whenNoDelegateCall {
+        _defaultParams.createWithTimestamps.shape = "this name is longer than 32 bytes";
+        vm.expectRevert(abi.encodeWithSelector(Errors.SablierLockup_ShapeNameTooLong.selector, 33, 32));
```

Comment:
similarly to fuzz, should we:
```solidity
string memory longShapeName = "this shape name is longer than 32 bytes";
_defaultParams.createWithTimestamps.shape = longShapeName;
vm.expectRevert(
    abi.encodeWithSelector(Errors.SablierLockup_ShapeNameTooLong.selector, bytes(longShapeName).length, 32)
);
```

Reply:
The shape name is fixed: "this name is longer than 32 bytes" in concrete and its length if fixed too: 33. That's why having a `33` here instead of `bytes(longShapeName).length` also tests that name length is calculated correctly. This was the reason I used `33` here instead of calculating length on runtime.

Reply:
fair enough, let's keep the current version 👍